tests/conftest.py

```diff
@@ -19,7 +20,7 @@ def nlp():
 
 @pytest.fixture()
 def opbank_spec():
-    with open("tests/resources/op_spec.json") as f:
+    with open("tests/r" "esources/op_spec.json") as f:
```

Comment:
Is it an error?

Reply:
@aby2s . I am not sure how it got chaged. Yeah! this is an error.